content/new-in-22.md

```diff
@@ -315,9 +340,26 @@ Make it easier for users to operate functionality through various inputs beyond
 
 Make Web pages appear and operate in predictable ways.
 
-### 3.2.6 Consistent Help (A)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#supermarketassistant">Supermarket assistant</a> with cognitive disabilities:</p>
+### [AG version] 3.2.6 Consistent Help (A)
+
+<section class="brief">
+<p>In brief:</p>
+   <dl>
+          <dt>Objective</dt><dd>Consistently locate user help</dd>
+          <dt>Author task</dt><dd>If you provide a help mechanism, provide it in a consistent location</dd>
+          <dt>Key beneficiaries</dt><dd>Users with some cognitive disabilities</dd>
+      </dl>
+</section>
+
+### [EOWG draft] 3.2.6 Consistent Help (A)
+
+<p>In brief:</p>
+   <ul>
+      <li><p><span class="whatwhy">What to do: </span>If you provide help, put  it in the same place on all web pages and app windows.</p></li>
```

Comment:
Still fiddling around with this one...
```suggestion
      <li><p><span class="whatwhy">What to do: </span>Consistently locate any user help that appears on multiple pages.</p></li>
```

Reply:
From [Kris Anne](https://www.w3.org/2002/09/wbs/35532/wcag2_2_in_brief/results#xq8):
... People who need help are more likely to find it if it is in a consistent place.

Reply:
Put help in the same place when it appears on multiple pages